frontend/src/components/Dropdown/SingleSelect/SingleSelectProvider.tsx

```diff
@@ -148,7 +171,7 @@ export const SingleSelectProvider = ({
     if (inputAriaProp) return inputAriaProp
     let label = 'No option selected'
     if (selectedItem) {
-      label = `Option ${itemToValue(selectedItem)}, selected`
+      label = `Option ${itemToLabelString(selectedItem)}, selected`
```

Comment:
Should we remove the comma?

```suggestion
      label = `Option ${itemToLabelString(selectedItem)} selected`
```

Reply:
I think comma adds a pause in screen reader so we should keep it